src/file_process.cpp

```diff
@@ -0,0 +1,117 @@
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <fcntl.h>
+#include <errno.h>
+#include <regex>
+#include <bits/stdc++.h> 
+#include <boost/algorithm/string.hpp>
+
+#include "../include/file_process.hpp"
+
+#define BUF_SIZE 2048
+
+using namespace std;
+
+void output_file_process(char *fin, char *fout) {
+    // check if the read access to the input file is OK
+    if (access(fin, R_OK)) {
+        std::cout << "The file " << fin << " can not be read from, errno = " << errno << "\n";
```

Comment:
This line exceeds 80 characters; I would suggest to break this line. 
Also, please check the cout statement in line 54 (similar problem)

Reply:
solved